src/lib/locationUtils.js

```diff
@@ -25,6 +25,18 @@ export function isWithinRegion(test, region) {
   )
 }
 
+export function getBounds({start, end}) {
+  if (!start || !end) {return []}
```

Comment:
If there's either a start or and end, would we want this to return the bounds being that one element?